src/routes/gifs/unlike-gif.e2e.test.js

```diff
@@ -0,0 +1,89 @@
+const request = require('supertest')
+// const {expect} = require('chai')
+// const db = require("../../db")
+const {fixtures} = require('../../../test/utils')
+const app = require('../../app')
+
+describe('UNLIKE /gif', () => {
+	let user
+
+	before(async ()=>{
+		 user = await fixtures.insertUser() 
+	})
+	describe('Failure', () => {
+		let accessToken
+		before(async ()=>{
+			const body = { id: user.id, email: user.email }
+			accessToken = await fixtures.generateAccessToken(
+				{data : {user : body}}
+			)
+		})
+		it('should return 400 if postId is not a number', async () => {
+			const post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'gif'
+			})
+			await fixtures.insertPostLike({
+				userId : user.id , 
+				postId : post.id}) 
+			return request(app)
+				.delete(`/api/v1/gifs/sdsdsd/likes/${user.id}`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(400)
```

Comment:
we'll be testing the error response as well

its done by adding a second parameter here: example

```js
const expectedError = {
        "error": {
          "message": "userId must be a number"
        }
       "status": "failed"
      }

// and then in the expect, it will not be 

.expect(400, expectedError)

```

before doing this just pass an empty object `{}` as a second parameter to it like this `expect(400, {})` and see the test failure that comes.